components/Common/Search/States/WithPoweredBy.tsx

```diff
@@ -5,9 +5,11 @@ import { useTranslations } from 'next-intl';
 import { useTheme } from 'next-themes';
 import { useEffect, useState } from 'react';
 
+import { STORYBOOK_MODE_THEME } from '@/.storybook/constants';
```

Comment:
We shouldn't be mixing Storybook constants with Application constants. Make a new set of constants :)